editor/js/ui/editor.js

```diff
@@ -413,10 +422,10 @@ RED.editor = (function() {
                         }
                     } else {
                         console.log("Unknown type:", definition.defaults[d].type);
-                        preparePropertyEditor(node,d,prefix);
+                        preparePropertyEditor(node,d,prefix,definition.defaults);
```

Comment:
Are you sure ?
In the preparePropertyEditor method we have the following code:
preparePropertyEditor(node,property,prefix,definition)
and then
definition[property]
so it should remain definition.defaults I guess